qutebrowser/browser/webengine/webenginesettings.py

```diff
@@ -344,6 +353,16 @@ def _init_default_profile():
     default_profile.setter.init_profile()
     default_profile.setter.set_persistent_cookie_policy()
 
+    _qute_scheme_handler.install(default_profile)
+    _req_interceptor.install(default_profile)  # type: ignore
+    _download_manager.install(default_profile)
+    cookies.install_filter(default_profile)
+
+    # Clear visited links on web history clear
+    history.web_history.history_cleared.connect(default_profile.clearAllVisitedLinks)
+    history.web_history.url_cleared.connect(
+        lambda url, profile=default_profile: profile.clearVisitedLinks([url]))
```

Comment:
This should probably be moved to a `webenginehistory.init()` or so, but I can take care of that after this PR is in.